src/test/getarg_tests.cpp

```diff
@@ -172,24 +172,24 @@ BOOST_AUTO_TEST_CASE(boolargno)
     const auto bar = std::make_pair("-bar", ArgsManager::ALLOW_ANY);
     SetupArgs({foo, bar});
     ResetArgs("-nofoo");
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", true));
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", false));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", true));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", false));
 
     ResetArgs("-nofoo=1");
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", true));
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", false));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", true));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", false));
 
     ResetArgs("-nofoo=0");
-    BOOST_CHECK(m_args.GetBoolArg("-foo", true));
-    BOOST_CHECK(m_args.GetBoolArg("-foo", false));
+    BOOST_CHECK(m_local_args.GetBoolArg("-foo", true));
+    BOOST_CHECK(m_local_args.GetBoolArg("-foo", false));
 
     ResetArgs("-foo --nofoo"); // --nofoo should win
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", true));
-    BOOST_CHECK(!m_args.GetBoolArg("-foo", false));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", true));
+    BOOST_CHECK(!m_local_args.GetBoolArg("-foo", false));
 
     ResetArgs("-nofoo -foo"); // foo always wins:
-    BOOST_CHECK(m_args.GetBoolArg("-foo", true));
-    BOOST_CHECK(m_args.GetBoolArg("-foo", false));
+    BOOST_CHECK(m_local_args.GetBoolArg("-foo", true));
+    BOOST_CHECK(m_local_args.GetBoolArg("-foo", false));
```

Comment:
This diff could be a scripted-diff in a separate commit?

Reply:
Yes, will be addressed in the next push.